src/main/java/seedu/address/logic/commands/AddExamCommand.java

```diff
@@ -34,8 +34,8 @@ public class AddExamCommand extends Command {
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_ID + "000001 "
             + PREFIX_EXAM_NAME + "Math "
-            + PREFIX_DATE + "2024-04-10 "
-            + PREFIX_TIME + "14:00";
+            + PREFIX_DATE + "2024-05-27 "
+            + PREFIX_TIME + "09:00";
```

Comment:
Understand that this changed so that during the PE (19/4/2024) and marking by the Teaching team, the sample command won't fail